sections/azera_shop_ribbon_section.php

```diff
@@ -16,6 +16,9 @@
 	$azera_shop_ribbon_title = get_theme_mod( 'azera_shop_ribbon_title' );
 	$azera_shop_button_text = get_theme_mod( 'azera_shop_button_text' );
 }
+$ribbon_background = apply_filters( 'azera_shop_translate_single_string', $ribbon_background, 'Ribbon section - background image' );
+$azera_shop_ribbon_title = apply_filters( 'azera_shop_translate_single_string', $azera_shop_ribbon_title, 'Ribbon section - title' );
```

Comment:
And check with Bogdan Preda why the gallery didn't form http://wraith.themeisle.com/wraith/stefan-cotitosu/azera-shop/compare/development/gallery.html